tutorials/Web/README.md

```diff
@@ -1,12 +1,103 @@
 # Compose for Web
-**The API is not finalized, and breaking changes can be expected**
 
-### Content:
+Compose for Web allows you to build reactive user interfaces for the web in Kotlin, using the concepts and APIs of Jetpack Compose to express the state, behavior, and logic of your application.
+
+Compose for Web provides multiple ways of declaring user interfaces in Kotlin code, allowing you to have full control over your website layout with a declarative DOM API.
+
+## Tutorials:
 
 [Getting Started With Compose for Web](Getting_Started/README.md) - create and configure a simple project
 
-[Building web UI](Building_UI/README.md) - overview of the basic features Compose for Web provides to build web UIs 
+[Building web UI](Building_UI/README.md) - overview of the basic features Compose for Web provides to build web UIs
 
 [Handing Events](Events_Handling/README.md) - a short overview of Events handling with compose web
 
 [Style DSL](Style_Dsl/README.md) - about styling the composable components in web
+
+## What's included:
+
+### Module `compose.runtime`
+It provides fundamental building blocks of Compose's programming model and state management.
+`Compose for Web` uses Jetpack's runtime implementation.
```

Comment:
Compose's

Reply:
@olonho Do you mean there is a typo?
I'm not sure I understand where it is.

The description for `runtime` was verified with: https://developer.android.com/jetpack/androidx/releases/compose-runtime#kts

Reply:
or do you want to clarify the last sentence?

`Compose for Web` uses Jetpack's runtime implementation. -> `Compose for Web` uses Compose Jetpack's runtime implementation.

Reply:
Jetpack is set of Android libraries, using it in Web tutorial is misleading and incorrect.

Reply:
fixed